src/api/index.js

```diff
@@ -117,9 +117,10 @@ export class Fledge {
 		const { port1: receiver, port2: sender } = new MessageChannel();
 
 		try {
-			port.postMessage([ 'runAdAuction', [
+			const msg = JSON.parse(JSON.stringify([ 'runAdAuction', [
```

Comment:
this is a weird hack I found to deal with `MessageChannel` and `postMessage` [serialization error](https://stackoverflow.com/questions/42376464/uncaught-domexception-failed-to-execute-postmessage-on-window-an-object-co)